tests/Functional/ContentTendersRenderTest.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Functional;
+
+use Behat\Mink\Element\NodeElement;
+use Drupal\Component\Utility\Html;
+use Drupal\user\Entity\Role;
+use Drupal\user\RoleInterface;
+
+/**
+ * Tests that Call for tender content type renders correctly.
+ */
+class ContentTendersRenderTest extends ContentRenderTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'config',
+    'block',
+    'system',
+    'path',
+    'oe_theme_helper',
+    'oe_theme_content_tender',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Give anonymous users permission to view entities.
+    Role::load(RoleInterface::ANONYMOUS_ID)
+      ->grantPermission('view published skos concept entities')
+      ->save();
+  }
+
+  /**
+   * Tests that the Tender page renders correctly.
+   */
+  public function testTenderRendering(): void {
+    // Create a document for Tender documents.
+    $media_document = $this->createMediaDocument('call_for_tenders_document');
```

Comment:
Add test coverage for the document checking ecl styling starting from div.ecl-file__info.

Reply:
We ensure that `.ecl-file__info` exist in ContentRenderTestBase::assertMediaDocumentDefaultRender().

Reply:
Indeed, sorry.